### ISSUE TITLE
Two issues encountered testing Coot 1.1.10 (homebrew)

### ISSUE THREAD

**Issue by olibclarke** [pemsley/coot#146]
Hi Paul,

Just listing two (possibly unrelated) issues encountered using the latest homebrew build (1.1.10, brewsci-bio-coot) with the tutorial model and data:

1. Traceback at startup - not sure if this is normal:
```bash
Traceback (most recent call last):
  File "/Users/oc2188/.config/Coot/coot_preferences.py", line 17, in <module>
    coot.show_modelling_toolbar()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'coot' has no attribute 'show_modelling_toolbar'
```

2. Ramachandran plot very glitchy: Here is a screen recording demonstrating what I mean: https://www.dropbox.com/scl/fi/qpvfmjqzuo01minjgf3hn/rama_bug.mov?rlkey=phn14jdm6gwkyuckhxx3rxow6&dl=0

Basically, the Rama plot loads ok, and residues are clickable (yay!). It does affect window resizing (making vertical resizing of the main window impossible), but otherwise it works. However, if I then try to close the rama plot, everything goes bananas - it initially closes, but then randomly reappears, the main window eventually becomes unresponsive and window resizing then causes a black screen. This is on a Macbook pro, apple silicon, Sonoma 14.5.

**Comment by rbs-sci** [comment]
I get a hard lock on Sonoma 14.5 when opening the Ramachandran plot window. Need to kill Coot. 1.1.10.

Only output is below:
```
** (MacCoot:41293): WARNING **: 17:06:34.551: Reimplement update_validation(). The function should iterate over the std::map holding validation data for each active graph and recompute it, then trigger a redraw.
update_ramachandran_plot() 0 FIXME?
debug:: accept_moving_atoms:: GTK4 update: update rama plot for 0
ERROR:: in unset_fixed_during_refinement_udd() mol is null

** (MacCoot:41293): WARNING **: 17:06:34.552: Reimplement update_validation(). The function should iterate over the std::map holding validation data for each active graph and recompute it, then trigger a redraw.
update_ramachandran_plot() 0 FIXME?
INFO:: run_post_manipulation_hook_py() No Python error on callable check

(MacCoot:41293): Gdk-CRITICAL **: 17:06:46.109: _gdk_macos_surface_update_fullscreen_state: assertion 'GDK_IS_MACOS_SURFACE (self)' failed

(MacCoot:41293): Gtk-WARNING **: 17:06:46.115: Broken accounting of active state for widget 0x12713c4d0(GtkCellView)

(MacCoot:41293): Gtk-WARNING **: 17:06:46.115: Broken accounting of active state for widget 0x12713ab10(GtkBox)

(MacCoot:41293): Gtk-WARNING **: 17:06:46.115: Broken accounting of active state for widget 0x12713a990(GtkToggleButton)

(MacCoot:41293): Gtk-WARNING **: 17:06:46.115: Broken accounting of active state for widget 0x12713a7e0(GtkBox)

(MacCoot:41293): Gtk-WARNING **: 17:06:46.115: Broken accounting of active state for widget 0x12713a660(GtkComboBox)

(MacCoot:41293): Gtk-WARNING **: 17:06:46.115: Broken accounting of active state for widget 0x12713a270(GtkGrid)

(MacCoot:41293): Gtk-WARNING **: 17:06:46.115: Broken accounting of active state for widget 0x12713a0f0(GtkBox)

(MacCoot:41293): Gtk-WARNING **: 17:06:46.115: Broken accounting of active state for widget 0x127138320(GtkDialog)
resize gl rama to 800 800
DEBUG:: --- new_startup_on_glarea_resize() 2674 2264
```

**Comment by pemsley** [comment]
> ERROR:: in unset_fixed_during_refinement_udd() mol is null

That looks like it might be related.

**Comment by pemsley** [comment]
> Traceback (most recent call last):
>   File "/Users/oc2188/.config/Coot/coot_preferences.py", line 17, in <module>
>     coot.show_modelling_toolbar()
>     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
> AttributeError: module 'coot' has no attribute 'show_modelling_toolbar'

This was mentioned on the mailing list I think. I have removed much old python toolbar code. We are looking at an old config and a new executable. Hopefully just removing line 17 from your `coot_preferences.py` will fix the problem.

**Comment by olibclarke** [comment]
Hi Paul,

I commented out these lines (17 plus the other ones referencing the modeling tool bar). This gets rid of the startup error pertaining specifically to the modeling toolbar, but all the Rama plot glitches persist.

Cheers
Oli

**Comment by rbs-sci** [comment]
> > ERROR:: in unset_fixed_during_refinement_udd() mol is null
> 
> That looks like it might be related.

Happened using the tutorial data, sorry, should have specified.

**Comment by pemsley** [comment]
> https://www.dropbox.com/scl/fi/qpvfmjqzuo01minjgf3hn/rama_bug.mov

Eeek! Poor Coot! Some of that might be GTK4 on macOS. Some is my doing.

I don't treat Coot like that!

OK, I'll see if I can reproduce it.

"/*/" means all atoms in the MODEL with number "*". "//" means all atoms in all models (as does "/", FWIW).

**Comment by olibclarke** [comment]
In my hands there is not much required to reproduce - just open coot, load tutorial data, launch rama plot, close it, then try resizing the window - everything glitches and eventually the screen goes black

**Comment by pemsley** [comment]
OK, I can more or less reproduce this.

**Comment by olibclarke** [comment]
Just noting that these glitches persist with the rama plot in Coot 1.1.12 & MacOS 15.3.1
